hwdb.d/60-keyboard.hwdb

```diff
@@ -1030,7 +1030,7 @@ evdev:input:b0003v17EFp6009*
  KEYBOARD_KEY_09001d=zoom                               # Fn+Space
  KEYBOARD_KEY_090011=prog1                              # ThinkVantage button
  KEYBOARD_KEY_090015=camera                             # Fn+F6 headset/camera VoIP key  ??
- KEYBOARD_KEY_090010=f20                                # Microphone mute button; should be micmute
+ KEYBOARD_KEY_090010=micmute                            # Microphone mute button
```

Comment:
This one interestingly enough actually is mapped to F20 at the kernel level. This was done by me actually, I'm afraid that for a while I simply used the F20-F23 values directly in the kernel. Which in hindsight was not a good idea.

Either way this one should be kept to correct the kernel mapping this to F20.

, so the old line

Reply:
let's keep it in for a bit, until somebody fixes the kernel?

Reply:
Ack.